### PR TITLE
[Ready] Improved danger output: show file name and line number

### PR DESCRIPTION
**Pull request by KrauseFx**
Nicer output

screenshot 2016-03-17 17 55 48

### CODE REVIEW

File: Dangerfile [hunk 1]
```diff
@@ -10,7 +10,8 @@ end
 
 require 'json'
 
-rspec_files = Dir[File.join(ENV["CIRCLE_ARTIFACTS"], "rspec_logs_*.json")]
+containing_dir = ENV["CIRCLE_ARTIFACTS"] || "." # for local testing
```
Comment (lacostej) [inline Dangerfile]: could the setting to "."by default cause unexpected issues (e.g.) if you run inside a different CI environment, etc.

Also, I noticed that I tried to use circleci to build my own branch of fastlane and it started to send information to some connected services. I.e. it doesn't differentiate YOUR circleci config, with mine.

At some point we should perhaps depend on a more explicit environment config.

File: spaceship/spec/portal/device_spec.rb [hunk 2]
```diff
@@ -27,7 +27,7 @@
     expect(device.platform).to eq('ios')
     expect(device.status).to eq('c')
     expect(device.model).to eq('iPhone 5 (Model A1428)')
-    expect(device.device_type).to eq('iphone')
+    expect(device.device_type).to eq('asdf')
```
Comment (lacostej) [inline spaceship/spec/portal/device_spec.rb]: this is obviously the test code, right ?

File: Dangerfile [hunk 3]
```diff
@@ -18,9 +19,14 @@ rspec_files.each do |current|
...
 
-    message = current_test["exception"]["message"].strip.gsub(/\n+/, "").gsub(/\\t+/, "").gsub(/\\n+/, "")[0..40]
-    error_message = "#{current_test["full_description"]}: #{message}"
-    puts error_message
-    fail("<code>#{error_message}</code>")
+    new_line = "<br />"
+    border = "<hr />"
+    message = current_test["exception"]["message"].strip.gsub(/\n+/, new_line).gsub(/\\t+/, new_line).gsub(/\\n+/, new_line)
+
+    error_message = "<code>#{current_test["file_path"]}:#{current_test["line_number"]}</code>"
```
Comment (lacostej) [inline Dangerfile]: consider adding the current gem path to improve location.

Comment (KrauseFx) [inline Dangerfile]: Good idea

Comment (KrauseFx) [inline Dangerfile]: Implemented :+1:

File: Dangerfile (hunk 1 above)
Comment (KrauseFx) [inline Dangerfile]: Is that really an issue? Did it fail your build? If the GitHub token is missing the might just fail and not even get to this point, right?

**Comment by KrauseFx** [comment]
Ready for Review :tada:

**Comment by asfalcone** [comment]
LGTM!

**Comment by KrauseFx** [comment]
:tada:

**Pull request merged**